apps/ima/views/confirm.html

```diff
@@ -1,3 +1,4 @@
+<title>{{#t}}pages.confirm.header{{/t}} – GOV.UK</title>
 {{<partials-page}}
   {{$page-content}}
     <table class="table-details">
```

Comment:
Add `<h1 class="gov-uk-heading-l">{{#t}}pages.confirm.header{{/t}}</h1>` above the table

Reply:
<h1 class="gov-uk-heading-l">{{#t}}pages.confirm.header{{/t}}</h1>  added